packages/core/src/editor/components/image-menu/image-bubble-menu.tsx

```diff
@@ -69,7 +66,7 @@ export function ImageBubbleMenu(props: EditorBubbleMenuProps) {
                 editor
                   ?.chain()
                   .focus()
-                  .setLogoAttributes({ size: value as AllowedLogoSize })
+                  .updateLogoAttributes({ size: value as AllowedLogoSize })
```

Comment:
The idea is that those `updateXXXAttributes` functions are defined on the node extension config, so they can be overridden in the Dashboard code, which will allow us to hook in and insert the `aliasFor` attribute to the node.